library/src/ui/index.js

```diff
@@ -85,4 +85,29 @@ window.CoNDeT.ui = {
       this.removeChild(childToRemove);
     },
   },
+
+  DisplayMode: (function () {
+    function constructor() {}
+
+    constructor.prototype.updateXY = function (x, y) {
+      this.setDeltaXY({ x, y });
+    };
```

Comment:
No `this` tutaj wskakuje na Mode, więc nie bardzo. Należy taką metodę dodać do `DisplayComponent` i przyda się jeszcze `setSeltaXY` (komentarz niżej). No i wtedy wywołać ją na `DisplayComponent`